nucypher/policy/identity.py

```diff
@@ -310,11 +310,13 @@ def lookup(cls, identifier: str, card_dir: Optional[Path] = CARD_DIR) -> Path:
             nickname, _id = identifier.split(cls.__DELIMITER)
         except ValueError:
             nickname = identifier
-        for filename in os.listdir(Card.CARD_DIR):
-            if nickname.lower() in filename.lower():
-                break
-        else:
+        filenames = [f for f in os.listdir(Card.CARD_DIR) if nickname.lower() in f.lower()]
+        if len(filenames) == 0:
```

Comment:
```suggestion
        if not filenames:
```
Perhaps a bit more idiomatic